LICENSE-binary

```diff
@@ -280,6 +280,10 @@ org.scala-lang:scala-reflect
 org.slf4j:jcl-over-slf4j
 org.yaml:snakeyaml
 org.rocksdb:rocksdbjni
+com.fasterxml.jackson.module:jackson-module-scala
+com.fasterxml.jackson.core:jackson-databind
+com.fasterxml.jackson.core:jackson-annotations
+com.fasterxml.jackson.core:jackson-core
```

Comment:
LICENSE-binary reflects the final content of the binary release tarball, so the parameter should be retained here.

Reply:
> LICENSE-binary reflects the final content of the binary release tarball, so the parameter should be retained here.

what is `the parameter`?

Reply:
sorry, the paranamer